axel/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Yong-hyu, Ban <yhban@quendi.moe>
+# Maintainer: Eon Jeong <administrator@korea.ac.kr>
+pkgname=axel
+pkgver=2.16.1
+pkgrel=1
+pkgdesc="Light command line download accelerator for Linux and Unix"
+arch=('i686' 'x86_64')
+license=('GPL')
+url="https://github.com/axel-download-accelerator/axel"
+groups=()
+depends=('openssl-devel' 'gettext-devel')
```

Comment:
devel packages can be only in makedepends. In depends you need write binary packages